client/extensions/woocommerce/app/store-stats/store-stats-chart/index.js

```diff
@@ -111,7 +112,11 @@ class StoreStatsChart extends Component {
 									selected={ tabIndex === selectedTabIndex }
 									tabClick={ this.tabClick }
 								>
-									<span className="store-stats-chart__value value">{ itemChartData.value }</span>
+									<span className="store-stats-chart__value value">
+										{ ( tab.type === 'currency' )
+											? formatCurrency( itemChartData.value )
```

Comment:
Here is an example (from our dashboard view):

* Load general settings https://github.com/Automattic/wp-calypso/blob/84a6f74ef22945ff7a5ed26b635a40de0925e67d/client/extensions/woocommerce/app/dashboard/manage-orders-view.js#L56
* Get currency object https://github.com/Automattic/wp-calypso/blob/84a6f74ef22945ff7a5ed26b635a40de0925e67d/client/extensions/woocommerce/app/dashboard/manage-orders-view.js#L149
* The currency object is passed to `ProcessOrdersWidget`, which uses `formatCurrency`: https://github.com/Automattic/wp-calypso/blob/84a6f74ef22945ff7a5ed26b635a40de0925e67d/client/extensions/woocommerce/components/process-orders-widget/index.js#L17-L28

> I had presumed that formatCurrency() was aware of the current store currency settings.

Nope, `formatCurrency` is a part of core Calypso and just defaults to USD, but you can pass it other currency codes. We really should have a wrapper around it that pulls in the correct currency (like we do on the [`PriceInput`](https://github.com/Automattic/wp-calypso/blob/82524bf32455d4340bda9d9cdd258dbc8e0e4d35/client/extensions/woocommerce/components/price-input/index.js) version of this) -- but it doesn't exist right now.